flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -174,6 +179,14 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         case "resumeAppHangTracking":
             resumeAppHangTracking(result)
 
+        case "sendReplayForEvent":
+#if canImport(UIKit) && !SENTRY_NO_UIKIT && (os(iOS) || os(tvOS))
```

Comment:
`|| os(tvOS)` 

we also support replay for tvos?

Reply:
no (it's not even supported by Flutter officially AFAIK), the condition is exactly the same as in cocoa core code